### PR TITLE
Fixed Issue #840 and added stars back

### PR DESCRIPTION
**Pull request by Alexandre2006**
## Status

Ready for review / in progress

## Description of Changes

Fixes #842 

Changes proposed in this pull request:

 - Fixed Issue 840
 - Added stars back

## Tests and linting

 - [x] I have rebased my changes on current `develop`

 - [x] pytests pass in the development environment on my local machine

 - [x] `flake8` checks pass

### CODE REVIEW

**Comment by Alexandre2006** [comment]
Hey could anyone rerun the travis-ci check because it failed for no reason. It just wasn't able to download a file from the internet. The code is right, it just probably got disconnected.

File: OpenOversight/app/main/forms.py [hunk 1]
```diff
@@ -294,8 +294,8 @@ class AddImageForm(Form):
 
 
 class DateFieldForm(Form):
-    date_field = DateField('Date <span class="text-danger">*</span>', validators=[DataRequired()])
-    time_field = TimeField('Time', validators=[Optional()])
+    date_field = DateField('Date*', validators=[DataRequired()])
+    time_field = TimeField('Time*', validators=[Optional()])
```
Comment (abandoned-prototype) [inline OpenOversight/app/main/forms.py:298]: The Time field is optional, so there shouldn't be a star on that line

**Review by abandoned-prototype — COMMENTED** [review]
Thanks @Alexandre2006, appreciate your work. There is one more small comment. And as an FYI, you don't have to start a new pull request when you want to change something, you can just change it on the branch you used for the existing pull request and the PR will update automatically

**Comment by Alexandre2006** [comment]
No problem. Sorry about that, I'll fix it tommorow morning.

**Comment by Alexandre2006** [comment]
Hi! I fixed the Time* issue and replace it to Time because Time is an optional answer. Please check it out.

**Review by abandoned-prototype — APPROVED** [review]
Great! Thanks for this @Alexandre2006

**Pull request merged**